integrations/cognito/gitbook-manifest.yaml

```diff
@@ -0,0 +1,31 @@
+name: cognito
+title: AWS Cognito
+icon: ./assets/icon.png
+previewImages:
+    - ./assets/preview.png
+description: Control who has access to your published documentation with AWS Cognito
+
+visibility: private
```

Comment:
Maybe publish as private until we have full marketing assets then public + verify it once it's ready.

Reply:
We have all the marketing assets. But I agree with keeping it private until we've tested it and it looks good!